### ISSUE TITLE
Why is validation loss calculated without entropy weight? 

### ISSUE THREAD

**Issue by mazatov** [wvangansbeke/Unsupervised-Classification#114]
Hello authors, I noticed that validation loss does not include entropy weight. That makes the training and validation loss very different and hard to compare for things like overfitting. I wanted to ask what is the reason behind not including entropy weight in validation?

```
@torch.no_grad()
def scan_evaluate(predictions):
    # Evaluate model based on SCAN loss.
    num_heads = len(predictions)
    output = []

    for head in predictions:
        # Neighbors and anchors
        probs = head['probabilities']
        neighbors = head['neighbors']
        anchors = torch.arange(neighbors.size(0)).view(-1,1).expand_as(neighbors)

        # Entropy loss
        entropy_loss = entropy(torch.mean(probs, dim=0), input_as_probabilities=True).item()
        
        # Consistency loss       
        similarity = torch.matmul(probs, probs.t())
        neighbors = neighbors.contiguous().view(-1)
        anchors = anchors.contiguous().view(-1)
        similarity = similarity[anchors, neighbors]
        ones = torch.ones_like(similarity)
        consistency_loss = F.binary_cross_entropy(similarity, ones).item()
        
        # Total loss
        total_loss = - entropy_loss + consistency_loss
        
        output.append({'entropy': entropy_loss, 'consistency': consistency_loss, 'total_loss': total_loss})

    total_losses = [output_['total_loss'] for output_ in output]
    lowest_loss_head = np.argmin(total_losses)
    lowest_loss = np.min(total_losses)

    return {'scan': output, 'lowest_loss_head': lowest_loss_head, 'lowest_loss': lowest_loss}
```

**Comment by wvangansbeke** [comment]
This is just to have an indication. You can add a weight if you want.

**Comment by mazatov** [comment]
@wvangansbeke Gotcha, thanks! 👍
